src/scorecard.js

```diff
@@ -0,0 +1,90 @@
+class Scorecard {
+  constructor() {
+    this._pinsKnocked = []
+    this._frame = 0
+    this._roll = 0
+    this._frameScores = []
+    this._strikesSpares = {}
+    this._maxFrames = 9
+  };
+
+  inputRoll(number) {
```

Comment:
Could have a return statement to the user/player, about the number pins they have knocked down.